src/wasm-lib/tests/executor/inputs/pattern_vase.kcl

```diff
@@ -0,0 +1,32 @@
+// Defines a vase.
```

Comment:
This is my rewrite of https://gist.github.com/adamchalmers/f1a89627d7343ae1d713761fb7f35f1e -- it operates MUCH faster.